src/gui/torrentcontenttreeview.cpp

```diff
@@ -107,6 +108,78 @@ void TorrentContentTreeView::keyPressEvent(QKeyEvent *event)
     }
 }
 
+void TorrentContentTreeView::setupDownloadPriorityMenu(QMenu *menu, const bool createSubMenu) const
+{
+    const auto model = qobject_cast<TorrentContentFilterModel *>(this->model());
+    Q_ASSERT(model);
+
+    const auto getSelectedRows = [this]() { return selectionModel()->selectedRows(TorrentContentModelItem::COL_PRIO); };
+
+    const auto applyPriorities = [&](const BitTorrent::DownloadPriority priority)
```

Comment:
please list the capture vars explicitly.

Reply:
Fixed by 251eb8331eb1b00057ded6d8089e257de9b09ff7